app/renderer/js/main.ts

```diff
@@ -994,6 +994,10 @@ class ServerManagerView {
 				webview.send('set-idle');
 			});
 		});
+
+		ipcRenderer.on('open-network-settings', () => {
+			this.openSettings('Network');
```

Comment:
I think you'll need to take care that the back button isn't enabled around here. This is what I get in the settings page when I click on your button:

![image](https://user-images.githubusercontent.com/24617297/65622468-55b4e700-dfe3-11e9-8748-33b0d3bda791.png)

Try playing around with `canGoBackButton()` in `webview.ts` and either enabling that button to go back to the new server form or disabling it entirely. I'd prefer the first option. @akashnimare what do you think?

Reply:
@kanishk98 I have tried for the different possibilities with the back button which are present in the following [link](https://drive.google.com/file/d/1Yo4Aytco1DccpxvnLCOBtWbGUYaOpdLf/view). However, I feel the back button behaves in a buggy manner as it starts working only after reloading the page. I feel the initLeftSidebarEvents() activates all the sidebar keys on initiating the app. So we don't have to activate or link the sidebar buttons again in every function. Please correct me if I am wrong.

Reply:
@akashnimare @kanishk98 Pls have a look and correct me if I am wrong.